widgets/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@getgreenspark/widgets",
-  "version": "1.4.1",
+  "version": "1.4.2-2",
```

Comment:
What is the purpose of the `-2` part? Is it for defining a pre-release, or metadata?

Reply:
<img width="642" alt="image" src="https://github.com/user-attachments/assets/a2c2f751-c5d0-4289-8b6f-5ed0e90aa5d1">